skbio/diversity/beta/_base.py

```diff
@@ -44,15 +47,25 @@ def pw_distances(counts, ids=None, metric="braycurtis"):
 
     See Also
     --------
+    unweighted_unifrac
+    weighted_unifrac
     scipy.spatial.distance.pdist
     pw_distances_from_table
 
     """
+    # this should probably go somewhere else. where?
+    _skbio_metrics = {'unweighted_unifrac': unweighted_unifrac,
+                      'weighted_unifrac': weighted_unifrac}
     num_samples = len(counts)
     if ids is not None and num_samples != len(ids):
         raise ValueError(
             "Number of rows in counts must be equal to number of provided "
             "ids.")
+    if metric in _skbio_metrics:
```

Comment:
oh, i see, the assumption is that if it isn't an skbio metric that scipy may know how to deal with it